### PR TITLE
Add clusterlogforwarder to forward app-logs to kafka

### PR DESCRIPTION
**Pull request by 4n4nd**
Related https://github.com/operate-first/apps/issues/232

This will forward app-logs to kafka topic `zero-prod.cluster.app-logs`
Update max message size bytes for that topic to 10MB ([slack discussion on size](https://coreos.slack.com/archives/CB3HXM2QK/p1617720645002700))
More docs on log forwarding to kafka: https://docs.openshift.com/container-platform/4.7/logging/cluster-logging-external.html#cluster-logging-collector-log-forward-kafka_cluster-logging-external

### CODE REVIEW

File: cluster-logging/overlays/moc/zero/clusterlogforwarders/clusterlogforwarder.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+apiVersion: logging.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+  name: instance
```
Comment (HumairAK) [inline cluster-logging/overlays/moc/zero/clusterlogforwarders/clusterlogforwarder.yaml:5]: Can this be named to something less generic? like `kafka-log-forwarder` maybe?

Comment (4n4nd) [reply cluster-logging/overlays/moc/zero/clusterlogforwarders/clusterlogforwarder.yaml:5]: This is required to be `instance` ([docs](https://docs.openshift.com/container-platform/4.7/logging/cluster-logging-external.html#cluster-logging-collector-log-forward-kafka_cluster-logging-external))

Comment (HumairAK) [reply cluster-logging/overlays/moc/zero/clusterlogforwarders/clusterlogforwarder.yaml:5]: Nevermind I see that it has to be `instance`:

From the docs: 
> The namespace for the ClusterLogForwarder CR must be openshift-logging.

**Comment by HumairAK** [comment]
As per the docs: 

> If a ClusterLogForwarder object exists, logs are not forwarded to the default Elasticsearch instance, unless there is a pipeline with the default output. 

We'll need to add the appropriate inputrefs to the default output so that current es ingestion is not effected.

**Pull request merged**